### PR TITLE
Added new streamlit-extras `Jupyterlite` 

### PR DESCRIPTION
**Pull request by Rjchauhan18**
I have added new streamlit extra -> `jupyterlite` as per https://github.com/arnaudmiribel/streamlit-extras/issues/133

Hello @arnaudmiribel 

For testing I have tryed to follow the step according to this [contribution guidelines](https://arnaudmiribel.github.io/streamlit-extras/contributing/) but my gallery/streamlit_app.py keeps running ... . 

So, can you please review it ? but i am sure that code is fine.

### CODE REVIEW

**Comment by Rjchauhan18** [comment]
Hello @arnaudmiribel 
Can you please review it?

File: src/streamlit_extras/jupyterlite/__init__.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        src="https://jupyterlite.github.io/demo/repl/index.html?kernel=python&toolbar=1", 
+        height=height,
+        width=width
+    )
+
+def example():
+    jupyterlite(1500,1600)
+
+__title__ = "Jupyterlite"  # title of your extra!
+__desc__ = "Add jupiterlite to your Streamlit app in one command"  # description of your extra!
```
Comment (arnaudmiribel) [inline src/streamlit_extras/jupyterlite/__init__.py:18]:
```suggestion
__desc__ = "Add a Jupyterlite sandbox to your Streamlit app in one command" 
```

File: src/streamlit_extras/jupyterlite/__init__.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+def example():
+    jupyterlite(1500,1600)
+
+__title__ = "Jupyterlite"  # title of your extra!
+__desc__ = "Add a Jupyterlite sandbox to your Streamlit app in one command" 
+__icon__ = "💡"  # give your extra an icon!
+__examples__ = [example]  # create some examples to show how cool your extra is!
+__author__ = "Rahul Chauhan"
+__experimental_playground__ = False  # Optional
```
Comment (arnaudmiribel) [inline src/streamlit_extras/jupyterlite/__init__.py:22]:
```suggestion
__title__ = "Jupyterlite"
__desc__ = "Add a Jupyterlite sandbox to your Streamlit app in one command" 
__icon__ = "💡"
__examples__ = [example]
__author__ = "Rahul Chauhan"
__experimental_playground__ = False
```

File: src/streamlit_extras/jupyterlite/__init__.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
+from streamlit.components.v1 import iframe
+
+from .. import extra
+
+@extra
+def jupyterlite(height:int,width:int):
```
Comment (arnaudmiribel) [inline src/streamlit_extras/jupyterlite/__init__.py:6]: Please add a docstring to this function. Look at other extras for inspiration and layout! Thanks ☺️

**Review by arnaudmiribel — CHANGES_REQUESTED** [review]
Thanks for the contribution! Please add a docstring and apply proper linting (using pre-commit) to your file, and that should do.

**Review by arnaudmiribel — COMMENTED** [review]
Nice job on passing the linting tests! Only remaining thing is please add a docstring to `jupyterlite` :-) you can look at other extras to get inspired. We use the Google docstring style. Example:

```python
@extra
def style_metric_cards(
    background_color: str = "#FFF",
    border_size_px: int = 1,
    border_color: str = "#CCC",
    border_radius_px: int = 5,
    border_left_color: str = "#9AD8E1",
    box_shadow: bool = True,
) -> None:
    """
    Applies a custom style to st.metrics in the page

    Args:
        background_color (str, optional): Background color. Defaults to "#FFF".
        border_size_px (int, optional): Border size in pixels. Defaults to 1.
        border_color (str, optional): Border color. Defaults to "#CCC".
        border_radius_px (int, optional): Border radius in pixels. Defaults to 5.
        border_left_color (str, optional): Borfer left color. Defaults to "#9AD8E1".
        box_shadow (bool, optional): Whether a box shadow is applied. Defaults to True.
    """
```

So here would be something like:

```python
@extra
def jupyterlite(height: int, width: int):
    """
    Add a Jupyterlite sandbox to your Streamlit app
    
    Args:
        width (int): Width of the Jupyterlite sandbox
        height (int): Height of the Jupyterlite sandbox
    """

    iframe(
        src="https://jupyterlite.github.io/demo/repl/index.html?kernel=python&toolbar=1",
        height=height,
        width=width,
    )
```

**Review by arnaudmiribel — APPROVED** [review]

**Pull request merged**